docs/control_center_docs/servo_module_getting_started.rst

```diff
@@ -0,0 +1,111 @@
+   holding its position at 0 displacement
+
+6. Now, set *Angle* to 3.14. You'll notice that the module will spin counterclockwise for about one second (with our Maximum Angular Velocity at 3.14 rad/s), 
+   then stop and hold the position
+7. Now, set *Maximum Angular Velocity* to 6.28 rad/s, and set *Angle* back to 0. You'll notice the module returns to its original position, this time getting there in about half a second
+8. Set *Coast* to release the module's hold
+
+********************
+Trajectory Control
+********************
+
+Vertiq's servo firmware provides a method of minimum jerk position planning through trajectory control. The module can hold a 
+queue of up to 32 commanded trajectories to complete depending on the *Trajectory Queue Mode* parameter available in the General tab.
+
+.. image:: ../_static/control_center_pics/servo_getting_started/servo_traj_queue_mode.png
+
+If *Trajectory Queue Mode* is set to *Append*, a trajectory sent while a trajectory is in progress will append to the end of the queue, 
+and will be completed immediately after the previous trajectory completes. If *Queue Mode* is set to *Overwrite*, a trajectory sent while a 
+trajectory is in progress will immediately take over, and the trajectory in progress is stopped. More information about both trajectories 
+in general as well as trajectory queue modes can be found :ref:`here <control_trajectory>`.
+
+.. note::
+    You must set a trajectory displacement before setting either an average speed or duration. Failure to do so will result in an invalid trajectory command, and no 
+    motion will occur.
+
+To test trajectory controls with the Control Center:
+
+1. Connect your module with the Control Center
+2. In the General tab, set *Maximum Angular Velocity* to 50 rad/s
+3. In the Testing tab, set *Angle* to 0 to make sure we are starting from 0 radians of displacement
+4. In the General tab, set *Trajectory Queue Mode* to *Append*
+5. Navigate to the Testing tab, and find the Trajectory testing parameters
+
+.. image:: ../_static/control_center_pics/servo_getting_started/servo_traj_testing.png
+
+6. For this example, we will set only a duration and displacement. Set *Trajectory Angular Displacement* to 12.56 rad
+7. Set *Trajectory Duration* to 15s, and you will see the module start to spin
+8. While the module is still spinning, set *Trajectory Angular Displacement* to 0 rad, and *Trajectory Duration* to 1s. This will queue a second trajectory 
+   bringing the module back to its initial position
+
+*******************************
+Communication Configuration
+*******************************
+Vertiq's servo firmware supports all supported :ref:`hobby_protocol`. Your module's target communication protocol can be configured in the 
+General tab under the *Communication* parameter. This parameter can be Standard PWM, a OneShot protocol, MultiShot, or a DShot protocol.
```

Comment:
The listing of protocols may be redundant since you link to the hobby protocol page that covers that in more detail